app/views/layouts/_menu.html.erb

```diff
@@ -4,17 +4,17 @@
       <li class="dropdown">
         <%= menu_dropdown_link(t(:administrate)) %>
         <ul class="dropdown-menu">
-          <% if can? :manage, Vote %>
+          <% if can? :index, Vote %>
```

Comment:
What was wrong with manage?

I gave a user the permission `:manage, :vote_user`, making it able to display this  menu. This item also became visible, probably because the user was allowed to index `/voteringar` (without the admin namespace)